### PR TITLE
Fixes #26 - Add aarch64 profile and Docker files

### LINKED ISSUE #26: Add profile for cross-compile for aarch64
Let's add a profile to be able to cross-compile for aarch64 the same as we do for other netty projects

### PR DESCRIPTION
**Pull request by martin-g**

### CODE REVIEW

**Comment by martin-g** [comment]
At the moment it fails with:
```
cross-compile-aarch64-build_1          | [INFO] ------------------------------------------------------------------------
cross-compile-aarch64-build_1          | [INFO] BUILD FAILURE
cross-compile-aarch64-build_1          | [INFO] ------------------------------------------------------------------------
cross-compile-aarch64-build_1          | [INFO] Total time:  5.946 s
cross-compile-aarch64-build_1          | [INFO] Finished at: 2020-12-08T09:20:50Z
cross-compile-aarch64-build_1          | [INFO] ------------------------------------------------------------------------
cross-compile-aarch64-build_1          | [ERROR] Failed to execute goal on project netty-incubator-transport-native-io_uring: Could not resolve dependencies for project io.netty.incubator:netty-incubator-transport-native-io_uring:jar:0.0.3.Final-SNAPSHOT: Could not find artifact io.netty:netty-transport-native-unix-common:jar:linux-aarch_64:0.0.3.Final-SNAPSHOT in sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
cross-compile-aarch64-build_1          | [ERROR] 
```
because of #22

**Comment by normanmaurer** [comment]
@martin-g can you sign our ICLA:

https://netty.io/s/icla ?

**Comment by martin-g** [comment]
Done!

File: pom.xml [hunk 1]
```diff
@@ -423,7 +481,159 @@
...
+          </plugin>
+
+          <plugin>
+            <groupId>org.fusesource.hawtjni</groupId>
+            <artifactId>maven-hawtjni-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>build-native-lib</id>
+                <configuration>
+                  <name>netty_transport_native_epoll_aarch_64</name>
```
Comment (normanmaurer) [inline pom.xml:581]:
```suggestion
                  <name>netty_quiche_linux_aarch_64</name>
```

File: pom.xml [hunk 2]
```diff
@@ -423,7 +481,159 @@
...
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+                <configuration>
+                  <archive>
+                    <manifest>
+                      <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                    </manifest>
+                    <manifestEntries>
+                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_epoll_aarch_64.so; osname=Linux; processor=aarch_64,*</Bundle-NativeCode>
```
Comment (normanmaurer) [inline pom.xml:618]:
```suggestion
                      <Bundle-NativeCode>META-INF/native/libnetty_quiche_linux_aarch_64.so; osname=Linux; processor=aarch_64,*</Bundle-NativeCode>
```

**Review by normanmaurer — CHANGES_REQUESTED** [review]

**Comment by martin-g** [comment]
I've just noticed https://github.com/netty/netty/pull/10845
Let me update to 4.1.55 and see how it goes!

**Comment by martin-g** [comment]
@normanmaurer All should be good now!

**Comment by normanmaurer** [comment]
There were still a few small issues... I fixed these and opened https://github.com/netty/netty-incubator-transport-io_uring/pull/29.

**Pull request closed without merging**